packaging/centos63/cloud.spec

```diff
@@ -129,6 +129,8 @@ Requires: perl
 Requires: libvirt-python
 Requires: qemu-img
 Requires: qemu-kvm
+Requires: genisoimage
```

Comment:
you'll need to add this to other cloud.spec files in packages/{centos7,fedora20,fedora21}